ethereum_history_api/circuits/lib/src/verifiers/tx_idx.nr

```diff
@@ -0,0 +1,25 @@
+use dep::u2b::u64_to_u8;
+use dep::proof::{rlp::decode0, utils::memcpy};
+use crate::misc::{arrays::sub_array_equals_up_to_length, bytes::{nibbles_to_bytes, left_to_right_padding, right_pad}};
+
+global KEY_LEN = 6;
```

Comment:
I know it's not part of this PR but maybe we could rename it to KEY_NIBBLE_LEN to avoid confusion?